src/utils/hooks/transaction/utils/extrinsic.ts

```diff
@@ -0,0 +1,129 @@
+import { ExtrinsicData } from '@interlay/interbtc-api';
```

Comment:
Can you document somewhere what is the full process of adding new extrinsic - which files need modification? So we are able to follow the full design easily with future changes?